src/CLHEPrandomgenerator.cpp

```diff
@@ -37,6 +37,7 @@
 
 #include "CLHEPrandomgenerator.h"
 #include <CLHEP/Random/Randomize.h>
+#include <CLHEP/Random/Random.h>
```

Comment:
```
/mnt/build/davidlt/starlight/a/slc6_amd64_gcc493/external/clhep/2.2.0.4-kpegke/include/CLHEP% grep -n -R 'createInstance' *
Random/Random.h:150:  static int createInstance();
Random/Randomize.h:70:static const int HepRandomGenActive = HepRandom::createInstance();
```

There is not other version of `createInstance ` where it would take `seed`.